### PR TITLE
130 fabs

### PR DESCRIPTION
**Pull request by simon-schmitz**
Addressing #130
- new pr with adapted branch naming
- old pr: https://github.com/helmholtz-analytics/heat/pull/224
- all comments from the old pr have been implemented

### CODE REVIEW

**Comment by krajsek** [comment]
Cannot find the branch in the repository. Please check in.
Currently there are availabe:
  master
  remotes/origin/47-pypi-deployment
  remotes/origin/HEAD -> origin/master
  remotes/origin/bug/198-unittests-failing
  remotes/origin/feature/177-unique
  remotes/origin/features/110-split_output_randn
  remotes/origin/features/163-split-reductions
  remotes/origin/features/168-right_hand_side_operators
  remotes/origin/features/216-version-attribute
  remotes/origin/features/227-lshape
  remotes/origin/features/98-moments
  remotes/origin/features/98-moments_2
  remotes/origin/features/convolve
  remotes/origin/features/global-setters+getters
  remotes/origin/master

File: heat/core/tests/test_rounding.py [hunk 1]
```diff
@@ -64,6 +64,54 @@ def test_ceil(self):
         with self.assertRaises(TypeError):
             ht.floor(object())
 
+    def test_fabs(self):
```
Comment (krajsek) [inline heat/core/tests/test_rounding.py]: as absolute==abs==fabs you might combine the tests for these synonyms.

File: heat/core/tensor.py [hunk 2]
```diff
@@ -556,6 +556,23 @@ def __eq__(self, other):
         """
         return relations.eq(self, other)
 
+    def fabs(self, out=None):
+        """
+        Calculate the absolute value element-wise and return floating-point tensor.
```
Comment (krajsek) [inline heat/core/tensor.py]: Please provide a short description explaining why we implement fabs, abs and absolute

**Comment by Markus-Goetz** [comment]
Bump, needs a merge of the master (file names and proper places might have changed)

**Pull request merged**